lib/widgets/side_pane.dart

```diff
@@ -1,14 +1,17 @@
-import 'package:collection/collection.dart';
+import 'package:files/widgets/context_menu/context_menu.dart';
+import 'package:files/widgets/context_menu/context_menu_entry.dart';
 import 'package:files/widgets/workspace.dart';
 import 'package:flutter/material.dart';
 
 class SidePane extends StatefulWidget {
   final List<SideDestination> destinations;
   final WorkspaceController workspace;
+  final void Function(String tabPath) onNewTab;
```

Comment:
Consider creating a typedef for this? Could improve code quality in multiple ways + reusability yay!

Reply:
I can create `typedef void StringCallback(String data);` inside `package:files/backend/utils.dart` for more reusability

Reply:
Creating the typedef inside this same file could be better, more organized (also maybe name it something like NewTabCallback)